geniza/corpus/management/commands/import_data.py

```diff
@@ -294,8 +307,57 @@ def add_document_language(self, doc, row):
             doc.language_note = "\n".join(notes_list)
             doc.save()
 
+    def import_document(self, row, joins, docstats, recto_verso_lookup):
```

Comment:
some of these could become class attributes if you'd like this cleaned up?

Reply:
ooh, I see my dev instructions about refactoring were vaguer than I thought! I did not want a method to import a single document, I wanted a method to import a set of documents. The inner loop of the current handle documents method.

 I thought the logic separation would be cleaner that way. Since you've worked on it, you may have a better sense about that?  Either way, this seems like a lot of stuff to be passing around — they should probably become class attributes if we need to pass them between methods.

Reply:
* I'm inclined to keep it as is, just because I feel like it'd be easier to test, but it shouldn't be hard to include the for loops. Let me know what you think.
* I'll move `joins` and `docstats` to the class, I don't see a way to refactor to get rid of those.
    * Making `joins` a class variable also allows us to place all the join logic in its own function. Let me know if you think that'd be a good idea.